cvat/apps/iam/exceptions.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework.exceptions import PermissionDenied
+
+class LimitsReachedError(PermissionDenied):
+    default_personal_detail = "You've reached the maximum number of {}. Contact the administrator to extend the limits."
+    default_org_detail = "You've reached the maximum number of {}. Contact the administrator to extend the limits for organization."
```

Comment:
Probably we can add here the slug for an organization

 ```suggestion
    default_org_detail = "You've reached the maximum number of {}. Contact the administrator to extend the limits for `{}` organization."
```